driver/bpf/fillers.h

```diff
@@ -572,36 +572,72 @@ FILLER(sys_poll_x, true)
 #define MAX_IOVCNT 32
 
 static __always_inline int bpf_parse_readv_writev_bufs(struct filler_data *data,
-						       const struct iovec __user *iovsrc,
+						       const void __user *iovsrc,
 						       unsigned long iovcnt,
 						       long retval,
 						       int flags)
 {
 	const struct iovec *iov;
+#if defined(CONFIG_X86_64)
```

Comment:
These ifdefs are needed because sometime the `compat_iovec` struct is not defined (e.g. on the `scap-open` test arm ci runners).